Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertGeometry.cs

```diff
@@ -325,7 +325,7 @@ public Curve NurbsToSpeckle(DB.NurbSpline revitCurve, Document doc, string units
     speckleCurve.degree = revitCurve.Degree;
     //speckleCurve.periodic = revitCurve.Period;
     speckleCurve.rational = revitCurve.isRational;
-    speckleCurve.closed = RevitVersionHelper.IsCurveClosed(revitCurve);
+    speckleCurve.closed = RevitVersionHelper.IsCurveCurveClosed(revitCurve);
```

Comment:
Typo? `CurveCurveClosed`?

Reply:
yah copy and paste 😁 pasted in wrong doc oops